sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/README.md

```diff
@@ -4,6 +4,7 @@ Azure Object Anchors enables an application to detect an object in the physical
 
 [Source code](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src) | [Package (NuGet)](https://www.nuget.org/packages/Azure.MixedReality.ObjectAnchors.Conversion/)
 
+
```

Comment:
Why an extra newline here?

This file is not related to DeviceUpdate. Revert?

Reply:
I have updated the code.